source-code/plugins/i18next/src/plugin.test.ts

```diff
@@ -1,8 +1,9 @@
 import { expect, it, describe } from "vitest"
-import { plugin, type PluginOptions } from "./plugin.js"
-import { Variant, createMockEnvironment, getVariant } from "@inlang/plugin"
+import { plugin} from "./plugin.js"
+import type { PluginOptions } from "./options.js"
+import { type Variant, createMockEnvironment, getVariant } from "@inlang/plugin"
 
-describe("plugin options", () => {
+describe("option pathPattern", () => {
 	it("should throw if the path pattern does not include the {language} placeholder", async () => {
```

Comment:
@inlang/plugins change `{language}` to `{languageTag}`?

Reply:
Done ✅